.env

```diff
@@ -5,4 +5,6 @@ DATABASE_USERNAME=username
 DATABASE_PASSWORD=secretpassword
 DATABASE_NAME=dbname
 JWT_SECRET=jwt_awesome_key
+JWT_SECRET_KEY=jwt_awesome_key
+JWT_EXPIRATION_TIME=3600
```

Comment:
that was somehow needed to get the jwt running.

The `JWT_SECRET` and `JWT_SECRET_KEY` needed to co-exist for whatever reason.